<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui_gui::gui</class>
 <widget class="QWidget" name="ui_gui::gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FindChangePhase</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCustomPlot" name="shortSigma" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="longSigma" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="vse" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>700</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>311</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_f0">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Несущая частота: (f0)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="f0">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_fdisk">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Частота дискретизации: (fdisk)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="fdisk">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_SNR">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>ОСШ: (snr)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="SNR">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_time">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Время: (t)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="t">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_duration">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Длительность: (duration)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="duration">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_bitrate">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Скорость передачи данных: (bitrate)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="bitrate">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_outputT">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Нечто: (outputT)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="outputT">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_p">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Нечто: (p)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="p">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_koef_dur">
           <property name="maximumSize">
            <size>
             <width>237</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Величина сдвига: (koef_dur)</string>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="koef_dur">
           <property name="maximumSize">
            <size>
             <width>63</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_graf">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="whiteGauss">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Condensed</family>
           </font>
          </property>
          <property name="text">
           <string>белый Гауссов шум</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="FlikerShum">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Condensed</family>
           </font>
          </property>
          <property name="text">
           <string>Розовый шум</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="raceNoise">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Condensed</family>
           </font>
          </property>
          <property name="text">
           <string>Райсовский шум</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="getShortSigma">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Condensed</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Get Short Sigma</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="getLongSigma">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Condensed</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Get Long Sigma</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calculate">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Condensed</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
